app/models/models.scala

```diff
@@ -12,7 +14,7 @@ case class Talk(
   url: String,
   authors: Seq[Author],
   location: String,
-  date: DateTime,
+  date: String,
```

Comment:
If https://github.com/guardian/digital-department-website/pull/9/files/336c432d1fd4cbe2c9e0beb6ce449bf9a8697aa9#diff-b5453a2f36ae98c317791919ee53fb30R39 is in scope, then that should cover that.